pentasharp/ViewModel/TaxiBooking/ConfirmBookingViewModel.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.ViewModel.TaxiBooking
+{
+    /// <summary>
+    /// Represents the data required to create a taxi booking.
+    /// </summary>
+    public class CreateBookingViewModel
```

Comment:
move this viewmodel and the others in this folder to a TaxiRequests folder you'll create under Models. I dont think ViewModel folder separation is necessary. I proposed something similar to Olta's PR https://github.com/starlabspro-internship/pentasharp/pull/24/files#r1867294136